src/test/text_document_test.py

```diff
@@ -0,0 +1,115 @@
+import unittest
+
+from docx import Document
+
+
+from data_ingestion.text_document import TextDocument
+
+
+class TextDocumentTest(unittest.TestCase):
+    def build_test_document(paragraphs):
+        doc_name = "my document"
+
+        docx_doc = Document()
+
+        # here we build the test document
+        for p in paragraphs:
+            docx_doc.add_paragraph(p)
+
+
+        test_filename = "test_docx_import.docx"
+
+        docx_doc.save(test_filename)
+
+        return TextDocument.from_docx(file_path = test_filename, name = doc_name)
+
+    def test_that_it_builds_from_a_word_document(self):
+        doc_name = "my document"
+        paragraphs = [
+            "[AP]: this is what was said _00:00:05_ and then this _00:01:23_"
+        ]
+        docx_doc = Document()
+
+        # here we build the test document
```

Comment:
Wouldn't you use `build_test_document()` here?

Reply:
good call

Reply:
I had a closer look at that test case needs access to the:
- docx document
- name used to create the document
The reason I didn't use the helper there is that I didn't want to over complicate the API for one (exhaustive) test case that touches something other than paragraphs.